kafka/producer.go

```diff
@@ -0,0 +1,59 @@
+// Package kafka provides producer and consumer to work with kafka topics
+package kafka
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"github.com/lvl484/positioning-filter/position"
+)
+
+type Producer interface {
+	Produce(position.Position) error
+	Close() error
+}
+
+type producer struct {
+	KafkaProducer sarama.SyncProducer
+	Config        *Config
+}
+
+func NewProducer(config *Config) (Producer, error) {
```

Comment:
In this case, that is not so important, but when some pointers are passed to a func you have to be sure that they are concurrently safe.